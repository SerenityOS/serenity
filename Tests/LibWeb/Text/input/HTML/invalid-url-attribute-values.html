<!DOCTYPE html>
<script src="../include.js"></script>
<script>
    test(() => {
        const elementList = [
            { "img": "currentSrc" },
            { "object": "data" },
        ];
        for (const elementDescriptor of elementList) {
            [elementName, propertyName] = Object.entries(elementDescriptor)[0];
            const element = document.createElement(elementName);
            println(`${elementName}.${propertyName} default value: '${element[propertyName]}'`);
        }
    });
</script>
