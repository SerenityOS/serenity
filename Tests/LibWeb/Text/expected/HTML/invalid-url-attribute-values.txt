img.currentSrc default value: ''
object.data default value: ''
